data/styles/application.css

```diff
@@ -16,6 +16,6 @@
  */
 
 .titlebar {
-    padding-bottom: 0;
-    padding-top: 0;
+    padding-bottom: 0.188rem;
+    padding-top: 0.188rem;
```

Comment:
Oh, duh! Smart. In this case I think we should leave a comment about where this is coming from for future spelunkers, though. Like "3px at 9pt font, see: https://github.com/elementary/stylesheet/blob/9b7122b07b3a1878501d13d842c54e83ef9061c6/src/gtk-3.0/_index.scss#L162" or something.